tripal_chado/includes/TripalImporter/GFF3Importer.inc

```diff
@@ -615,6 +623,9 @@ class GFF3Importer extends TripalImporter {
       $this->logMessage("Step  8 of 26: Processing !num_features features...               ",
           ['!num_features' => number_format(count(array_keys($this->features)))]);
       $this->insertFeatures();
+      $this->logMessage("Step  8.1 of 26: Processing !num_features feature Names to update...               ",
```

Comment:
I know this is a pain but can you change this to step 9 and increment those that follow?

Reply:
Oh Yes!! I meant to do that from the start and got over excited when it all worked!! Fixed.